gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigsResource.java

```diff
@@ -29,26 +30,35 @@
 import com.google.inject.name.Named;
 import com.linkedin.restli.common.ComplexResourceKey;
 import com.linkedin.restli.common.EmptyRecord;
+import com.linkedin.restli.common.HttpStatus;
 import com.linkedin.restli.server.CreateResponse;
 import com.linkedin.restli.server.UpdateResponse;
 import com.linkedin.restli.server.annotations.RestLiCollection;
 import com.linkedin.restli.server.resources.ComplexKeyResourceTemplate;
 
+import org.apache.gobblin.service.monitoring.FlowStatusGenerator;
+
+import static org.apache.gobblin.service.FlowStatusResource.FLOW_STATUS_GENERATOR_INJECT_NAME;
+
 
 /**
  * Resource for handling flow configuration requests
  */
 @RestLiCollection(name = "flowconfigs", namespace = "org.apache.gobblin.service", keyName = "id")
 public class FlowConfigsResource extends ComplexKeyResourceTemplate<FlowId, EmptyRecord, FlowConfig> {
   private static final Logger LOG = LoggerFactory.getLogger(FlowConfigsResource.class);
+  public static final String FLOW_CONFIG_GENERATOR_INJECT_NAME = "flowConfigsResourceHandler";
```

Comment:
Why is the name not matching the value? Is the idea that this resource handler is a config generator?

Reply:
Yes, same as FLOW_STATUS_GENERATOR_INJECT_NAME in FlowStatusResource.java